ui/desktop/src/components/settings/app/AppSettingsSection.tsx

```diff
@@ -402,7 +404,16 @@ export default function AppSettingsSection({ scrollToSection }: AppSettingsSecti
           </CardDescription>
         </CardHeader>
         <CardContent className="pt-4 px-4">
-          <div className="flex space-x-4">
+          <div className="flex flex-wrap gap-3">
+            <Button
+              onClick={() => setShowReportFailureModal(true)}
+              variant="default"
+              size="sm"
+              className="flex items-center gap-2"
+            >
+              <AlertCircle size={16} />
+              Report a Failure
+            </Button>
```

Comment:
I see that this is now next to a "report a bug" button. I think your button is better because it collects system info, so I'd say go ahead and remove the other "report a bug" button so we don't have two very similar actions